composer.json

```
@@ -84,7 +84,7 @@
         }
     },
     "scripts": {
-        "fix-cs": "php bin/php-cs-fixer fix --config=.php_cs.dist"
+        "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.dist.php"
```

```suggestion
        "fix-cs": "bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php"
```
To use the local `php-cs-fixer`

---

It should use the local `php-cs-fixer`: https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands.

---

Didn't know about that 👍